susePublicCloudInfoClient/pint

```diff
@@ -56,8 +56,8 @@ import sys
 
 from docopt import docopt
 
-import infoserverrequests
-import version
+import lib.susepubliccloudinfoclient.infoserverrequests as ifsrequest
+import lib.susepubliccloudinfoclient.version as version
 
```

Comment:
I'd say just have what works after install. During development one can either just install from the working directory or set the environment such that the local lib is found first. Setting PATH or PYTHON_PATH work.